yourgov/components/StaffMembersTable/ModalConfirmRemoveAccess.tsx

```diff
@@ -0,0 +1,69 @@
+import { useMemo, useState } from 'react';
+import { ButtonGroup, Button } from '@ag.ds-next/react/src/button';
+import { Text } from '@ag.ds-next/react/src/text';
+import { Modal } from '@ag.ds-next/react/src/modal';
+import { plural /* RowData */ } from './lib/utils';
+import { StaffMemberWithIndex } from './lib/types';
+
+type RowData = StaffMemberWithIndex;
+
+export type ModalConfirmRemoveAccessProps = {
+	isOpen: boolean;
+	onConfirm: () => void;
+	onClose: () => void;
+	itemsToDelete: RowData | RowData[];
+};
+
+export function ModalConfirmRemoveAccess({
+	isOpen,
+	onConfirm,
+	onClose,
+	itemsToDelete,
+}: ModalConfirmRemoveAccessProps) {
+	const [submitting, setSubmitting] = useState(false);
+
+	function onSubmit() {
+		setSubmitting(true);
+		setTimeout(() => {
+			setSubmitting(false);
+			onConfirm();
+		}, 2000);
```

Comment:
Should probably be consistent with the main form process
```suggestion
		}, 1500);
```